integration/tests/epochs/suite.go

```diff
@@ -417,8 +411,19 @@ func (s *Suite) SubmitStakingCollectionCloseStakeTx(
 	return result, nil
 }
 
-// SubmitAdminRemoveNodeTx will submit the admin remove node transaction
-func (s *Suite) SubmitAdminRemoveNodeTx(ctx context.Context,
+
+func (s *Suite) removeNodeFromProtocol(ctx context.Context, env templates.Environment, nodeID flow.Identifier)  {
+	result, err := s.submitAdminRemoveNodeTx(ctx, env, nodeID)
+	require.NoError(s.T(), err)
+	require.NoError(s.T(), result.Error)
+
+	// ensure we submit transaction while in epoch setup phase
+	currView := s.assertInPhase(ctx, flow.EpochPhaseStaking)
+	require.True(s.T(), stakingAuctionViews > currView, "expected remove node transaction to be completed in the epoch staking phase")
```

Comment:
Is the second assertion (line 422) to make sure we are in the _first_ epoch staking phase? Could add a comment if so

Reply:
I am removing since its only verifying if we are in first epoch staking auction